nephio-ansible-install/roles/docker/tasks/main.yaml

```diff
@@ -0,0 +1,29 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
+
+---
+- name: ensure repository key is installed
+  become: true
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+    state: present
+
+- name: ensure docker registry is available
+  become: true
+  apt_repository: repo='deb https://download.docker.com/linux/ubuntu bionic stable' state=present
+
+- name: ensure docker and dependencies are installed
+  become: true
+  apt: name=docker-ce update_cache=yes
+
+- name: Add user to docker group
+  become: true
+  shell: usermod -aG docker {{ cloud_user }}
+
+- name: Reset connection so docker group is picked up.
+  meta: reset_connection
+- name: Create the docker network for kind (to make our job easier plumbing up the lot)
```

Comment:
you will need a blank line after L25

Reply:
fixed: https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/395f7e39e539da55dfac934b347349d1f61f9f83